tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -0,0 +1,394 @@
+import { range } from 'common/collections';
+import { BooleanLike } from 'common/react';
+import { resolveAsset } from '../assets';
+import { useBackend } from '../backend';
+import { Box, Button, Icon, Stack } from '../components';
+import { Window } from '../layouts';
+
+const ROWS = 5;
+const COLUMNS = 6;
+
+const BUTTON_DIMENSIONS = '50px';
+
+type GridSpotKey = string;
+
+const getGridSpotKey = (spot: [number, number]): GridSpotKey => {
+  return `${spot[0]}/${spot[1]}`;
+};
+
+const CornerText = (props: {
+  readonly align: 'left' | 'right';
+  readonly children: string;
+}): JSX.Element => {
+  const { align, children } = props;
+
+  return (
+    <Box
+      style={{
+        position: 'relative',
+        left: align === 'left' ? '2px' : '-2px',
+        'text-align': align,
+        'text-shadow': '1px 1px 1px #555',
+      }}>
+      {children}
+    </Box>
+  );
+};
+
+type AlternateAction = {
+  icon: string;
+  text: string;
+};
+
+const ALTERNATE_ACTIONS: Record<string, AlternateAction> = {
+  remove_splints: {
+    icon: 'crutch',
+    text: 'Remove splints',
+  },
+
+  remove_accessory: {
+    icon: 'tshirt',
+    text: 'Remove accessory',
+  },
+
+  retrieve_tag: {
+    icon: 'tags',
+    text: 'Retrieve info tag',
+  },
+
+  toggle_internals: {
+    icon: 'mask-face',
+    text: 'Toggle internals',
+  },
+};
+
+const SLOTS: Record<
+  string,
+  {
```

Comment:
This should be its own type - there is a weird reference to this later on

Reply:
Done I think?